src/components/DivElement.js

```diff
@@ -1,6 +1,10 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
-
+class DivElement extends HTMLElement {
+    constructor(content) {
+      super('div', content);
+    }
+  }
 // Export class here
-module.exports = {};
+module.exports = {DivElement};
```

Comment:
Tests are failing, you'll want to assign the export to just `DivElement`, not in an object. Same goes for `HTMLElement` (See `Timer.js` for example).

Reply:
Updated and fixed. What a silly mistake.